into/backends/spark.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import division, print_function, absolute_import
+
+
+class Dummy(object):
+    pass
+
+try:
+    from pyspark import SparkContext
+    import pyspark
+    from pyspark import RDD
+    from pyspark.rdd import PipelinedRDD
+    from pyspark.sql import SchemaRDD, SQLContext
+    RDD.min
+except (AttributeError, ImportError):
+    SchemaRDD = PipelinedRDD = RDD = SparkContext = SQLContext = Dummy
+    pyspark = Dummy()
+
+
+from toolz import memoize
+from datashape import var
+
+from .. import convert, append, discover
+
+
+@append.register(SparkContext, list)
+def list_to_spark_context(sc, seq, **kwargs):
+    return sc.parallelize(seq)
+
+
+@append.register(SparkContext, object)
+def anything_to_spark_context(sc, o, **kwargs):
+    return append(sc, convert(list, o, **kwargs), **kwargs)
```

Comment:
If someone does the following

```python
sc = pyspark.SparkContext(...)
sql = pyspark.SQLContext(sc)

rdd = sc.parallelize(...)
srdd = into(sql, rdd)
```

Then I think that this will trigger an unnecessary collect/parallelize.